ArgoProject/src/components/Health.h

```diff
@@ -19,7 +19,7 @@ namespace app::comp
 	public: // Public Member Functions
 	public: // Public Static Variables
 	public: // Public Member Variables
-		int health;
+		unsigned int health;
```

Comment:
```suggestion
		std::uint32_t health;
```
Use the standard libraries number types for anything that isn't plain `int/float/double` as they are typedef'd to be guaranteed that size on all platforms.